tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-docker-release-pipeline.yml

```diff
@@ -0,0 +1,72 @@
+parameters:
+  - name: commit_sha1
+    displayName: Commit Sha1
+    type: string
+    default: master
+  - name: image_tag
+    displayName: Image Tag
+    type: string
+    default: latest
+
+stages:
+- stage: Python_Packaging
+
+  variables:
+    docker_image_prefix: onnxruntime-training
+    linux_gpu_dockerfile: dockerfiles/Dockerfile.training
+
+  jobs:
+  - job: Linux_py_GPU_Build_Test_Release_Dockerfile
+    timeoutInMinutes: 0
+    workspace:
+      clean: all
+    pool: Linux-GPU-CUDA10
+    steps:
+      - task: CmdLine@2
+        displayName: Build builder stage of docker file
+        inputs:
+          script: |
+            docker build \
+              --pull \
+              -t ${{ variables.docker_image_prefix }}-manylinux-gpu-release-stage1 \
+              --target builder \
+              --build-arg COMMIT_SHA1="${{ parameters.commit_sha1 }}" \
+              -f ${{ variables.linux_gpu_dockerfile }} .
+          workingDirectory: $(Build.SourcesDirectory)
+
+      - task: CmdLine@2
+        displayName: Run tests
+        inputs:
+          script: |
+            docker run \
+              --gpus all \
+              --rm \
+              ${{ variables.docker_image_prefix }}-manylinux-gpu-release-stage1 \
+              python onnxruntime/tools/ci_build/build.py \
+              --build_dir onnxruntime/build \
+              --config Release \
+              --test \
```

Comment:
@edgchen1 Will this run all end-to-end tests? Or are there some additional ones that need manual trigger?

Reply:
I think end-to-end test uses a different flag. Let me try that

Reply:
it won't run all e2e tests. they are run nightly, but can also be triggered manually. perhaps we can update the e2e test builds to use the docker image produced by this build.

Reply:
No this doesn't run end-to-end tests. There's another task to consolidate nightly build runs. We can probably address variant tests there. One set of tests for build dockerfile, one set for release dockerfile.